CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 2.1.3
+
+* Fix: explicitly specify `'localhost'` for round trip tests
+* Remove unnecessary import of HTTP2 lib
+* Run `dartfmt`
```

Comment:
Keep the Changelog to user-relevant changes (IE. only the first item)

Reply:
Done.